release/ray_release/byod/build.py

```diff
@@ -204,9 +204,13 @@ def _validate_and_push(byod_image: str) -> None:
         .strip()
     )
     expected_ray_commit = _get_ray_commit()
-    assert (
-        docker_ray_commit == expected_ray_commit
-    ), f"Expected ray commit {expected_ray_commit}, found {docker_ray_commit}"
+    if os.environ.get("RAY_IMAGE_TAG"):
```

Comment:
seems that in this case you do not need to run `_get_ray_commit()`?

Reply:
good catch